DokuwikiXMLExport.php

```diff
@@ -100,17 +100,15 @@ private function getPageIds()
      */
     private function splitConfigToArray($config)
     {
-        $array = explode(',', $config);
-        $trimmedArray = array_map('trim', $array);
-        return $trimmedArray;
+        return preg_split('(\s*,\s*)', $config);
```

Comment:
The parentheses `(...)` are used for capturing group when doing regex matching. They serve no purpose when splitting.

Reply:
Done